docs/source/developer/add_test.rst

```diff
@@ -0,0 +1,93 @@
+
+##############
+add_test.cmake
+##############
+
```

Comment:
Is this generated with CMinx? If so don't check it in. If not, API documentation should use CMinx.

Reply:
They are generated with CMinx, I will remove them from the pull request.